### ISSUE TITLE
Visual Folder Error

### ISSUE THREAD

**Issue by D-A-R-K-N-E-S-S** [Rexabit/valheim-visuals-modifier#1]
When I installed your mod, the log was filled with many errors. It wasn't working and perhaps due to the errors, WackysDatabase ceased to function as well (all custom configs were ignored).

I found a particular error message in the log relating to your mod not being able to locate the config/Visuals folder. When I manually created this folder and restarted, everything worked perfectly (including WackysDatabase).

Long story short, your mod is not creating the "Visuals" folder when run for the first time and will not work until manually created.

Also, I want to say thank you - the mod is amazing.

**Comment by Rexabit** [comment]
Interesting, thought I fixed this before I published. Will double check and get it resolved.

**Comment by Rexabit** [comment]
This should now be resolved in version 0.0.2, which is now pushed to Thunderstore as well.
